extensions/adapters/raster/src/main/java/mil/gma/giat/geowave/gdal/InstallGdal.java

```diff
@@ -63,15 +117,15 @@ public static void main(
 			}
 		}
 		if (file.endsWith("zip")) {
-			TestUtils.unZipFile(
+			unZipFile(
```

Comment:
Can't create a dependency on TestUtils from here. Otherwise, the dependency becomes cyclic.